src/coreclr/pal/inc/pal.h

```diff
@@ -3452,7 +3437,6 @@ InterlockedDecrement(
     return result;
 }
 
-#define InterlockedDecrementAcquire InterlockedDecrement
```

Comment:
I would keep `InterlockedDecrementAcquire`. It mirrors `InterlockedDecrementRelease` and so it is odd to have one without the other.

Reply:
Sure, I will add it back. Should `InterlockedDecrementAcquire` use `InterlockedIncrement` or was decrement correct method?

Reply:
Actually, after looking at what all we have here - it is fine to delete it. We do not have a lot of consistency in what we have here. No action needed.

> Should InterlockedDecrementAcquire use InterlockedIncrement or was decrement correct method?

decrement was correct method.